### DISCUSSION TITLE
[Bug]: linux arm rpi5 not flashing t-beam

### DISCUSSION THREAD

**Discussion by jamieduk** [meshtastic/firmware#5129] [Troubleshooting]
### Category

Serial

### Hardware

T-Beam

### Firmware Version

any

### Description

[Bug]: linux arm rpi5 not flashing t-beam it just sais connecting when i select usb via brave-browser and i tried better chromium browser aswell via ubuntu arm 64 bit linux rpi 5 also how do i put in pairing mode with default pin of 12345678 as the led screen dont work ?

### Relevant log output

```Shell
all it sais is this and stays

esptool.js
Serial port WebSerial VendorID 0x1a86 ProductID 0x55d4
Connecting...
```

**Comment by jamieduk** [comment]
Any ideas???

**Reply by fifieldt** [reply]
1. Disconnect from USB
2. Hold down the middle button of t-beam
3. Connect USB
4. Wait a few seconds
5. Release Middle Button
6. Try flashing.

**Reply by fifieldt** [reply]
There's also now a "1200 baud reset" button in the flasher that does something similar.

**Comment by caveman99** [comment]
this requires web serial support. please check with your browser vendor if this is actually availabe on your platform.
